vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -136,12 +136,33 @@ EOF
 }
 
 enableMarinerKata() {
-    # Enable the mshv boot path
-    sudo sed -i -e 's@menuentry "CBL-Mariner"@menuentry "Dom0" {\n    search --no-floppy --set=root --file /EFI/Microsoft/Boot/bootmgfw.efi\n        chainloader /EFI/Microsoft/Boot/bootmgfw.efi\n}\n\nmenuentry "CBL-Mariner"@'  /boot/grub2/grub.cfg
+    echo "Contents of blkid output"
+    export my_blkid=$(blkid)
+    echo $my_blkid
+
+    # Add DOM0 boot entry above default
+    sudo sed -i -e 's@menuentry "CBL-Mariner"@menuentry "Dom0_legacy" {\n    search --no-floppy --set=root --file /EFI/Microsoft/Boot/bootmgfw.efi\n        chainloader /EFI/Microsoft/Boot/bootmgfw.efi\n}\n\nmenuentry "CBL-Mariner"@'  /boot/grub2/grub.cfg
+
+    SERVICE_FILEPATH="/etc/systemd/system/set-kataconfig.service"
+    touch ${SERVICE_FILEPATH}
+    cat << EOF > ${SERVICE_FILEPATH}
+[Unit]
```

Comment:
please avoid inline cat when possible - we primarily use it for runtime configuration to avoid adding artifacts to cloud init.

for build time, it should be possible to add file artifacts directly.

Reply:
@alexeldeib would you like us to turn this into a separate file (e.g. under scripts/linux/mariner) and add it via packer_source.sh to the UVM, or what was your suggested idea?

Reply:
Moved inline cat to a separate artifact file.